test/module/shared_model/bindings/BuilderTest.java

```diff
@@ -0,0 +1,145 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigInteger;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class BuilderTest {
+    static {
+        try {
+            System.loadLibrary("iroha");
+        } catch (UnsatisfiedLinkError e) {
+            System.err.println("Native code library failed to load. \n" + e);
```

Comment:
Is it a part of the test? I mean if the iroha lib not loaded test should be failed.

Reply:
There's exit(1) below that will fail test anyway. I'll replace it with an explicit assertion